### PR TITLE
TA-580 fix Task Description

### PR DESCRIPTION
**Pull request by boubaker**
Fix task description CKEditor instanciation by using default CKEditor configuration and style.

### CODE REVIEW

File: task-management/src/main/java/org/exoplatform/task/management/assets/javascripts/project-menu.js [hunk 1]
```diff
@@ -90,9 +90,17 @@ define('project-menu', ['SHARED/jquery', 'SHARED/taskLocale', 'ta_edit_inline',
           });
           
           
-          //CKEDITOR.basePath = '/commons-extension/ckeditor/';
+          CKEDITOR.basePath = '/commons-extension/ckeditor/';
           $dialog.find('textarea').ckeditor({
-            customConfig: '/task-management/assets/org/exoplatform/task/management/assets/ckeditorCustom/config.js'
+            customConfig: '/commons-extension/ckeditorCustom/config.js',
+            removePlugins: 'selectImage,simpleLink,hideBottomToolbar',
+            toolbarLocation: 'top',
```
Comment (plamarque) [inline task-management/src/main/java/org/exoplatform/task/management/assets/javascripts/project-menu.js]: I think toolbars are at the bottom currently.

Comment (boubaker) [reply task-management/src/main/java/org/exoplatform/task/management/assets/javascripts/project-menu.js]: In fact for Task Description, it's not. See it on 4.4 [here](http://plfentrial-4.4.1-training.acceptance7.exoplatform.org/portal/intranet/tasks).
But I can change it if you prefer, WDYT ?

File: task-management/src/main/java/org/exoplatform/task/management/assets/javascripts/project-menu.js [hunk 2]
```diff
@@ -90,9 +90,17 @@ define('project-menu', ['SHARED/jquery', 'SHARED/taskLocale', 'ta_edit_inline',
           });
           
           
-          //CKEDITOR.basePath = '/commons-extension/ckeditor/';
+          CKEDITOR.basePath = '/commons-extension/ckeditor/';
           $dialog.find('textarea').ckeditor({
-            customConfig: '/task-management/assets/org/exoplatform/task/management/assets/ckeditorCustom/config.js'
+            customConfig: '/commons-extension/ckeditorCustom/config.js',
```
Comment (thomasdelhomenie) [inline task-management/src/main/java/org/exoplatform/task/management/assets/javascripts/project-menu.js]: Not sure why it has not been changed as part of https://jira.exoplatform.org/browse/PLF-6630 . There must be a good reason but I can't remember which one. So let's move on.

Comment (boubaker) [reply task-management/src/main/java/org/exoplatform/task/management/assets/javascripts/project-menu.js]: From @trongtt (Room "Core developers")

> it is mainly about different UI

**Pull request merged**